python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -147,8 +147,13 @@ def to_datetime(
     if utc:
         raise NotImplementedError("utc is not yet implemented")
 
-    if format is not None and "%f" in format:
-        format = format.replace("%f", "%9f")
+    if format is not None:
+        if "%Z" in format or "%z" in format:
+            raise NotImplementedError(
+                "cuDF does not yet support timezone-aware datetimes"
+            )
+        elif "%f" in format:
+            format = format.replace("%f", "%9f")
```

Comment:
Note: This is fine, but I'll just note that `%%Z` is a valid string for `strptime` that means "look for a literal `%Z`". So this replacement (and the existing `%f` replacement) is not robust against some of the wackier formats a user might provide.